Lib/test/test_finalization.py

```diff
@@ -280,8 +280,9 @@ def test_simple_resurrect(self):
             gc.collect()
             self.assert_del_calls(ids)
             self.assert_survivors(ids)
-            # XXX is this desirable?
-            self.assertIs(wr(), None)
+            # This used to be None because weakrefs were cleared before
+            # calling finalizers.  Now they are cleared after.
+            self.assertIsNot(wr(), None)
```

Comment:
The changes to tests look to be taken from the initial commit.  I would like a comment or a test for the weakref with a callback.